tests/integ/test_large_query_breakdown.py

```diff
@@ -112,6 +112,46 @@ def test_no_valid_nodes_found(session, caplog):
     assert "Could not find a valid node for partitioning" in caplog.text
 
 
+def test_large_query_breakdown_external_cte_ref(session):
```

Comment:
do we have a test to test breakdown at cte node?

Reply:
added now